server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -544,6 +544,14 @@ export default class ServiceProviderReferralsController {
       Number(sessionNumber)
     )
 
+    if (req.method === 'POST') {
```

Comment:
I think that we've been using this pattern of one method for the `GET` and the `POST` in the cases where both of them render the same view. Here it's a bit different, since the `POST` doesn't have any functionality in common with the `GET`. Perhaps it's best to put it in a separate method? Also, I wonder whether the correct URL for the action would be `/service-provider/action-plan/:actionPlanId/appointment/:sessionNumber/post-session-feedback/submit`?

Reply:
Yep, posting to `check-your-answers` felt very weird, this makes a lot more sense!